packages/SwingSet/src/kernel/state/kernelKeeper.js

```diff
@@ -307,12 +307,42 @@ export default function makeKernelKeeper(
     kvStore.set('kernel.defaultReapInterval', `${defaultReapInterval}`);
   }
 
+  /**
+   *
+   * @param {string} mt
+   * @returns { asserts mt is ManagerType }
+   */
+  function insistManagerType(mt) {
+    assert(
+      [
+        'local',
+        'nodeWorker',
+        'node-subprocess',
+        'xs-worker',
+        'xs-worker-no-gc',
+      ].includes(mt),
+    );
+    return undefined;
```

Comment:
Won't a function return `undefined` anyway if you don't have an explicit `return` statement?  Or is this to deal with some kind of TypeScript nonsense?

Reply:
yeah, typescript

Reply:
That would be the difference between a return type of `void` vs `undefined`.

Reply:
let me know if there's a better way to write that assertion function, I just copied something else that works